blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -164,7 +165,6 @@
     ROLLUP_ON_READ_REPAIR_SIZE_PER_THREAD( "5" ),
     ROLLUP_ON_READ_TIMEOUT_IN_SECONDS("10"),
 
-    DELAYED_METRICS_MILLIS("300000"),
```

Comment:
If I am not mistaken, the other one is used by Tracker, to print/log whether or not a metric is delayed. I know it seems extra/redundant but I think we have used it in the past to just catch metrics that are delayed by certain time, without actually changing the actual ROLLUP_DELAY_MILLIS. 

I agree the name should be more clear that it's used by Tracker.